python_modules/libraries/dagster-dbt/dagster_dbt/include/dbt_scaffold/definitions.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+from dagster import Definitions
+
+from dagster_dbt import DbtCli, DbtManifest, dbt_assets
+
+DBT_PROJECT_DIR = Path(__file__).parent.joinpath("..", "jaffle_shop")
+DBT_MANIFEST_PATH = DBT_PROJECT_DIR.joinpath("target", "manifest.json")
+DBT_MANIFEST = DbtManifest.read(path=DBT_MANIFEST_PATH)
```

Comment:
Uppercasing this while not uppercasing `my_dbt_assets`, `schedules`, and `defs` feels weird. My instinct would be to make it lowercase, even if that means `DBT_PROJECT_DIR` and `DBT_MANIFEST_PATH` need to be lowercase as well for consistency (though could go either way on those two - feels different because they're strings I think). Thoughts?